WordPress/Classes/ViewRelated/Likes/LikesListDependency.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import WordPressKit
+
+
+/// A simple, convenient dependency wrapper for LikesListController.
+class LikesListDependency {
```

Comment:
I'm sorry, I don't quite understand the purpose of this separate class. All its content is only used by `LikesListController`, so why not just define them there?

Reply:
Moved the class into `LikesListController`. Thanks for the suggestion!